.github/workflows/spiced_docker_nightly.yml

```diff
@@ -1,12 +1,3 @@
-##################################################################################################################################
-#                                                                                                                                #
-#  WARNING: TOOL-GENERATED FILE - DO NOT EDIT MANUALLY!                                                                          #
-#                                                                                                                                #
-#  Command: yq 'explode(.)' .github/workflow_templates/spiced_docker_nightly.yml > .github/workflows/spiced_docker_nightly.yml   #
-#                                                                                                                                #
-#  Keep this notice at the top of the file.                                                                                      #
-#                                                                                                                                #
-##################################################################################################################################
```

Comment:
I've generated file as instructed from workflow, not sure why it removed the coment

`yq 'explode(.)' .github/workflow_templates/spiced_docker_nightly.yml > .github/workflows/spiced_docker_nightly.yml`

Reply:
It's a manual step to keep the comment